frontend/src/time_utils.js

```diff
@@ -20,5 +20,5 @@ export function formatDate(date) {
   if (day.length < 2) 
       day = '0' + day;
 
-  return [month, day, year].join('/');
+  return mFirst? [month, day, year].join('/') : [year, month, day].join('/');
```

Comment:
nit: make `/` a constant delimiter as it can be easily changed in the future?
since previously it was a `-`
maybe you can add it to your constants file?

Reply:
made it a constant. the  constants file I currently have is scoped to the management feature so I just made it in this file